src/Workspaces/CSharp/Portable/Simplification/Reducers/CSharpDefaultExpressionReducer.Rewriter.cs

```diff
@@ -45,13 +45,10 @@ public Rewriter(ObjectPool<IReductionRewriter> pool)
             }
 
             public override SyntaxNode VisitDefaultExpression(DefaultExpressionSyntax node)
-            {
-                return SimplifyNode(
+                => SimplifyNode(
                     node,
                     newNode: base.VisitDefaultExpression(node),
-                    parentNode: node.Parent,
                     simplifier: _simplifyDefaultExpression);
```

Comment:
this was the second part of hte bug.  passing in node.parent here was incorrect and meant we didn't see the impact of other changes we were making.